client/web/workflow/src/views/Workflow/List.vue

```diff
@@ -368,6 +403,30 @@ export default {
         })
         .catch(this.toastErrorHandler(this.$t(`notification:${event}.failed`)))
     },
+
+    statusText (w) {
+      return w.enabled ? this.$t('general:disable') : this.$t('general:enable')
+    },
+
+    statusClass (w) {
+      return w.enabled ? 'text-secondary' : 'text-primary'
+    },
+
+    handleStatusChange (w) {
+      const { workflowID } = w
+      this.$AutomationAPI.workflowRead({ workflowID })
+        .then(({ workflowID, enabled }) => {
+          this.$AutomationAPI.workflowUpdate({ ...w, workflowID, enabled: !enabled })
```

Comment:
think you misunderstood me here. The reason you fetch it is so you have the latest version and you only toggle the enabled.